content/id/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -0,0 +1,186 @@
+bagaimana cara untuk merutekan trafik internal. Ketika diaktifkan, EndpointSlice dapat memberikan peningkatan
+performa untuk Service yang memiliki Endpoint dalam jumlah besar.
+
+### Tipe-tipe Alamat
+
+EndpointSlice mendukung tiga macam tipe alamat:
+
+* IPv4
+* IPv6
+* FQDN (_Fully Qualified Domain Name_)
+
+### Topologi
+
+Setiap _endpoint_ pada EndpointSlice dapat memiliki informasi topologi yang relevan.
+Hal ini digunakan untuk mengindikasikan di mana _endpoint_ berada, mengandung informasi mengenai
+Node yang bersangkutan, zona, dan wilayah. Ketika nilai-nilai tersebut tersedia,
+label-label Topology berikut akan ditambahkan oleh _controller_ EndpointSlice:
+
+* `kubernetes.io/hostname` - Nama dari Node tempat _endpoint_ berada.
+* `topology.kubernetes.io/zone` - Zona tempat _endpoint_ berada. 
+* `topology.kubernetes.io/region` - Region tempat _endpoint_ berada.
+
+Nilai-nilai dari label-label berikut berasal dari sumber daya yang terasosiasi dengan tiap
+_endpoint_ pada sebuah _slice_. Label _hostname_ merepresentasikan nilai dari bidang NodeName
+ pada Pod yang bersangkutan. Label zona dan wilayah merepresentasikan nilai
+dari label-label dengan nama yang sama pada Node yang bersangkutan. 
+
+### Pengelolaan
+
+Secara _default_, EndpointSlice dibuat dan dikelola oleh _controller_
+EndpointSlice. Ada berbagai macam kasus lain untuk EndpointSlice, seperti
+implementasi _service mesh_, yang memungkinkan adanya entitas atau _controller_ lain
+yang dapat mengelola beberapa EndpointSlice sekaligus. Untuk memastikan beberapa entitas dapat
+mengelola EndpointSlice tanpa mengganggu satu sama lain, sebuah
+label `endpointslice.kubernetes.io/managed-by` digunakan untuk mengindikasikan entitas
+yang mengelola sebuah EndpointSlice. _Controller_ EndpointSlice akan menambahkan
+`endpointslice-controller.k8s.io` sebagai nilai dari label tersebut pada seluruh
+EndpointSlice yang dikelolanya. Entitas lain yang mengelola EndpointSlice juga diharuskan untuk
+menambahkan nilai yang unik untuk label tersebut.
+
+### Kepemilikan
+
+Pada kebanyakan kasus, EndpointSlice akan dimiliki oleh Service yang dilacaknya. Hal ini diindikasikan dengan referensi pemilik pada tiap EndpointSlice dan
+juga label `kubernetes.io/service-name` yang memudahkan pencarian seluruh
+EndpointSlice yang dimiliki oleh sebuah Service.
+
+## _Controller_ EndpointSlice
+
+_Controller_ EndpointSlice mengamati Service dan Pod untuk memastikan EndpointSlice
+yang bersangkutan selalu dalam kondisi terkini. _Controller_ EndpointSlice akan mengelola EndpointSlice untuk
+setiap Service yang memiliki _selector_. Ini akan merepresentasikan IP dari Pod
+yang cocok dengan _selector_ dari Service tersebut.
+
+### Ukuran EndpointSlice
+
+Secara _default_, tiap EndpointSlice terbatas dengan jumlah 100 _endpoint_ yang dapat dimiliki. Kamu dapat
+mengkonfigurasi hal ini dengan _flag_ `--max-endpoints-per-slice` {{< glossary_tooltip
```

Comment:
```suggestion
mengkonfigurasi hal ini dengan parameter `--max-endpoints-per-slice` {{< glossary_tooltip
```

what if we use parameter, instead of italicized flag?
paramter is registered in KBBI

Reply:
I think `opsi` would fit better for this case?

Reply:
:+1: for opsi

Reply:
Solved in https://github.com/kubernetes/website/pull/20166/commits/2e77f8a8acdbc6cb0cb503c32270c1b2871ae0e0